core/locale/domain/dateTime.go

```diff
@@ -17,6 +23,29 @@ func (dtf *DateTimeFormatter) SetDateTime(time time.Time, localtime time.Time) {
 	dtf.localDateTime = localtime
 }
 
+// SetLocation sets location for local time
+func (dtf *DateTimeFormatter) SetLocation(loc string) *DateTimeFormatter {
+	location, err := time.LoadLocation(loc)
+
+	if err != nil {
+		if dtf.logger != nil {
+			dtf.logger.WithField(flamingo.LogKeyMethod, "SetLocation").Error(
+				fmt.Sprintf("%s: failed to load time zone location: %v", loc, err),
```

Comment:
Why exactly logging here and skip to report the error to the callier? 
I would appreciate to report back the error occouring on setting an invalid time location - the caller could then decide to go on or handle/abort.

By just logging no code using this would recognize that SetLocation is not successful

Reply:
I updated the code to return an error. I left the logging with original error as error description also acts as an error code/key in FE context.